src/LibraryManager/CacheService/CacheService.cs

```diff
@@ -4,10 +4,10 @@
 using System;
 using System.Collections.Generic;
 using System.IO;
-using System.Runtime.InteropServices;
 using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.Web.LibraryManager.Contracts;
+using Microsoft.Web.LibraryManager.Utilities;
 
 namespace Microsoft.Web.LibraryManager
```

Comment:
is the convention to not use the folder name for public APIs?

Reply:
It originally was mixed, and hasn't had a cleanup pass to make it all consistent.